src/stack-hci/azext_stack_hci/generated/_params.py

```diff
@@ -43,6 +43,12 @@ def load_arguments(self, _):
                    'of the proxy resource holding details of HCI ArcSetting information.')
         c.argument('arc_instance_resource_group', options_list=['--instance-rg'], type=str, help='The resource group '
                    'that hosts the Arc agents, ie. Hybrid Compute Machine resources.')
+        c.argument('arc_application_client_id', type=str, help='App id of arc AAD identity.')
```

Comment:
The default value of `type` is `str`, please do not define `type=str` repeatedly